roxie/udplib/udptrs.cpp

```diff
@@ -440,214 +440,6 @@ class UdpReceiverEntry
 class CSendManager : public CInterface, implements ISendManager
 {
     friend class send_send_flow;
-    class CMessagePacker : public CInterface, implements IMessagePacker
```

Comment:
I moved this code out of the class with a view to using it to support streaming in localSlave mode - in the end I didn't use it, but I think it's cleaner as it is so I left the change in.